packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md

```diff
@@ -0,0 +1,166 @@
+Use the Devices SDK to add custom push verification functionality to your Android app.
+
+> **Note:** The following sample code assumes that suspend functions are called in a coroutine scope. See [Kotlin Coroutines](https://kotlinlang.org/docs/coroutines-overview.html) for more information.
+
+<!-- Data flow DIAGRAM -->
+
+### Install the dependency
+
+Add the Okta Devices SDK dependency to your `build.gradle` file:
+
+```kotlin
+implementation("com.okta.devices:devices-push:$okta.sdk.version")
+```
+
+where `$okta.sdk.version` is the latest release version. See [Release Status](https://github.com/okta/okta-devices-kotlin/releases) for the latest Okta Devices SDK version.
+
+### Initialize the client
```

Comment:
why are we repeating the same thing in developer doc instead of just pointing to github readme? If we do want to repeat it here we'll need to update code based on https://github.com/okta/okta-devices-kotlin/pull/8

Reply:
Hi @mingxiajiang-okta , @chrisdmills-okta provided an answer in the Slack thread. Does it work for you? https://okta.slack.com/archives/C01DR8G2QBE/p1659082811476939?thread_ts=1658854573.339039&cid=C01DR8G2QBE